iphone/Classes/TiUtils.h

```diff
@@ -460,6 +460,8 @@ typedef enum
 
 +(NSDictionary*)sizeToDictionary:(CGSize)size;
 
++(NSDictionary*)touchPropertysToDictionary:(UITouch*)touch:(CGPoint)point;
```

Comment:
Rename ``touchPropertysToDictionary`` to ``touchPropertiesToDictionary`` and replace all other missspellings.